docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,317 @@
+***************************************
```

Comment:
@peterschwarz I don't understand the reasoning on this. The code and spec are both updated to match. It seems that you are requesting that I create a situation where I have the spec and code out of sync.